modules/lib/lib-schema/src/test/resources/test/UpdateDynamicContentSchemaHandlerTest.js

```diff
@@ -21,4 +21,403 @@ exports.updateInvalidContentSchemaType = function () {
+        "type": "CONTENT_TYPE",
+        "form": [
+            {
+                "formItemType": "OptionSet",
+                "name": "radioOptionSet",
+                "label": "Single selection",
+                "expanded": false,
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9936826)
```suggestion
                'expanded': false,
```